lib/module.js

```diff
@@ -490,6 +490,10 @@ function tryModuleLoad(module, filename) {
   }
 }
 
+Module.import = function(request, parent) {
+  return ESMLoader.import(request, parent);
+};
```

Comment:
Why not make this `Module.prototype.import` so you can get rid of the `parent` parameter (and use `this` instead)?

Reply:
there eventually could be multiple loaders / I would like to see this fn removed when `import` lands

Reply:
`Module.import` doesn't appear to be used anywhere else in this PR, is it dead code?

Reply:
used as stop gap til `import()` in 6.2 lands

Reply:
> used as stop gap til import() in V8@6.2 lands

Is there a rough idea of when that'll be?
Would `Module.import` be intended as public consumable API?

Reply:
Sometime in Node@9
no, i'd remove it once 6.2 lands.

Reply:
We should guard this by checking for experimentalModules